### PR TITLE
Replace Home page with Landing page

### PR DESCRIPTION
**Pull request by hdenisenko**
resolves issue #70

### CODE REVIEW

File: web-app/src/components/App/index.js [hunk 1]
```diff
@@ -17,12 +16,11 @@ import * as ROUTES from '../../constants/routes';
 
 const App = () => (
     <Router>
-        <div class="main-container">
+        <div className="main-container">
```
Comment (hdenisenko) [inline web-app/src/components/App/index.js:19]: this was a console error unrelated to this ticket

File: web-app/src/components/Navigation/index.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ class Navigation extends Component {
           onMouseLeave={() => this.setState({ toggle: 0 })}
         >
           {links.map((link) => (
-            <li>
+            <li key={link[0]}>
```
Comment (hdenisenko) [inline web-app/src/components/Navigation/index.js:36]: also console error unrelated to the ticket

**Comment by hdenisenko** [comment]
also this Vercel check failed? Is this something I should set up on my end or is it not necessary?

**Comment by paul-rinaldi** [comment]
@hdenisenko To clarify, I would like the home button to still be a tab but the home button should take yo to the welcoming screen (and the links that are currently on the home page from the current home button in production should be placed in the footer). Thanks!

File: web-app/src/constants/navlinks.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import * as ROUTES from './routes';
 
 export const DEFAULTS = [
-  [ROUTES.HOME_PAGE, "Home"],
+  [ROUTES.LANDING, "Home"],
```
Comment (hdenisenko) [inline web-app/src/constants/navlinks.js:4]: added this tab back!

File: web-app/src/components/Footer/index.js [hunk 4]
```diff
@@ -16,6 +16,12 @@ const Footer = () => (
             <Link to={ROUTES.TERMS_AND_CONDITIONS}>
                 Terms and Conditions
             </Link>
+            <a href="https://211wisconsin.communityos.org/guided-search">
+                211 Madison Basic Needs Search
+            </a>
+            <a href="https://www.mkehometostay.org/">
+                Home to Stay
+            </a>
```
Comment (hdenisenko) [inline web-app/src/components/Footer/index.js:24]: moved these to the footer

**Pull request merged**